src/ifaddrs_posix.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// http://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+#[cfg(target_os = "android")]
+use get_if_addrs_sys::freeifaddrs;
```

Comment:
I'd suggest combining these lines into just
```
#[cfg(target_os = "android")] 
use get_if_addrs_sys::{freeifaddrs, getifaddrs, ifaddrs};
```
and same for the following:
```
#[cfg(not(target_os = "android"))]
use libc::{freeifaddrs, getifaddrs, ifaddrs};
```

Reply:
Good catch! Fixed.